lib/models/models.dart

```diff
@@ -6,8 +6,9 @@ class ProductModel extends HiveObject {
   final String name;
   final String picture;
   final double price;
+  final String id;
 
-  ProductModel({this.name, this.picture, this.price});
+  ProductModel({this.name, this.picture, this.price, this.id});
```

Comment:
I think we should store "Currency" as well, with SEEDS an option, and all our Fiat Currencies an option - e.g products could be priced in Fiat, and will then be converted to Seeds based on day rate when they are being used. 

I will approve because not sure if that should be a separate PR or not.

Reply:
Yeah, this PR is parity with what we have in place. But in firebase now instead of local storage. 

We can do a follow up PR to store Currency

Reply:
Story #361  Does not have AC about storing Currency. 

Wre can create a new story to handle this work.